### DISCUSSION TITLE
About GitReleaseManager bot

### DISCUSSION THREAD

**Discussion by dennisdoomen** [GitTools/GitVersion#3084] [General]
It's off-topic, but how did you guys produce these automatic links to a release? I found [this](https://github.com/GitTools/GitReleaseManager), but that seems to be a CLI tool for producing release notes, something that Github does even better these days. 

<img width="349" alt="image" src="https://user-images.githubusercontent.com/572734/163431759-9fa2b56e-0bed-4998-a242-a07573776f23.png">

**Comment by arturcic** [comment]
We use the mentioned tool as part of the release process. When we do a release all the issues in the milestone that we specify are fetched to generate the release notes, and also to send that message

**Comment by gep13** [comment]
GitReleaseManager, the application that you linked to, is a fairly opinionated tool that creates release notes based on issues/PRs that are assigned to a milestone, however, it can also take an input file of release notes. There are scriban templates that you can define to control exactly how the release notes are made up, if you don't like the default.

On top of this, it has the ability to manage the Release on GitHub, includes adding assets to the release, adding SHA's of the assets, closing the associated milestone and updating all issues associated with the milestone (the feature that you mentioned), and also to export the generated release notes.

The release notes that you can generate on GitHub directly (as far as I am aware) are generated from commit messages in the repository. This is something that GitReleaseManager doesn't do.

**Comment by dennisdoomen** [comment]
Thanks guys!
